seed/serializers/pint.py

```diff
@@ -94,6 +94,9 @@ def apply_display_unit_preferences(org, pt_dict):
     API and collapse any Quantity objects present down to a straight float, per
     the organization preferences.
     """
+    # skip non dictionary values
+    if not isinstance(pt_dict, dict):
+        return pt_dict
```

Comment:
Add a guard to prevent the function from attempting to recursively unit collapse the new labels id array.